tools/testing/target_determination/heuristics/interface.py

```diff
@@ -25,10 +25,10 @@
 @total_ordering
 class Relevance(Enum):
     HIGH = 4
-    NONE = 3
-    PROBABLE = 2
+    PROBABLE = 3
```

Comment:
lol, I have no idea why 3 changes from `NONE` to `PROBABLE`, and 0 changes from `UNRANKED` to `NONE`, any context here?

Reply:
It kind of undoes #112671 by making the traversal order go from most relevant -> least relevant instead of heuristics feels strongly (positive or negative) -> heuristics does not have an opinion.